performance_regression_gradual_grow_throughput.py

```diff
@@ -33,12 +33,29 @@ class Workload:
     wait_no_compactions: bool
     step_duration: str
     prepare_schema: bool
+    test_table: str = "keyspace1.standard1"
 
     def __post_init__(self):
         if isinstance(self.num_threads, int):
             # If only one thread count is provided, convert it to a list
             self.num_threads = [self.num_threads]
 
+        # Determine the stress tool and set the test table accordingly.
+        # This will be used for choosing correct post_prepare_cql_cmd
+
+        # TODO: improve this part with https://github.com/scylladb/qa-tasks/issues/2020
+        cs_cmd_tmpl = self.cs_cmd_tmpl[0] if isinstance(self.cs_cmd_tmpl, list) else self.cs_cmd_tmpl
+        stress_tool = cs_cmd_tmpl.split(" ")[0]
+        if stress_tool == "scylla-bench":
+            self.test_table = "scylla_bench.test"
+        elif stress_tool in ["cassandra-stress", "cql-stress-cassandra-stress"]:
+            self.test_table = "keyspace1.standard1"
+        elif stress_tool == "latte":
+            # https://github.com/scylladb/qa-tasks/issues/2020
+            pass
```

Comment:
I think would be good to create a warning SCT event saying something like `make sure that rune script disables speculative retries for its table.

Reply:
warning added